Tic_Tac_Toe_V2.1/test/BoardTest.java

```diff
@@ -81,7 +81,21 @@ public void methodGetCurrentBoard_whenGivenALargerNumberOfRowsThanColumns_return
     }
 
     @Test
-    public void methodCreateBoard_whenGivenNumberOfRowsAndColumns_returnsArrayOfStringsWithCorrectNumnberOfRowsAndColumns(){
+    public void methodCreateBoard_whenGivenNumberOfRowsAndColumns_returnsArrayOfStringsWithCorrectNumberOfRowsAndColumns(){
 
+        Board currentBoard = new Board(6, 6);
```

Comment:
I understand this test is trying to test `createBoard()` method.

could you pls tell me what `Board currentBoard = new Board(6, 6)` is for? :)

Reply:
its for testing if the board is still ok if given dimensions other than 3x3

Reply:
sorry Emily - I can't see that in the test (or I don't quite follow you). could you pls elaborate that more? 😞

Reply:
hmm i think maybe i don't understand what you're asking? might be easier to go over in person tomorrow

Reply:
oh hang on. you mean, what is Board currentBoard = new Board(6, 6). Its a new instantiation of object Board, with 6 rows and 6 columns given as arguments.